app/views/ServiceConfigPage.scala.html

```diff
@@ -55,7 +55,7 @@ <h1 id="service-header">Config: @serviceName</h1>
                                     <ul>
                                       @for(sourceValue: ConfigSourceValue <- envSources.get(env).getOrElse(Seq.empty)) {
                                         <li class="config-value" title="@ConfigService.friendlySourceName(sourceValue.source, env)">
-                                            <span class="truncate-content-overflow">@sourceValue.value</span>
+                                            <span class="truncate-content-overflow @if(sourceValue.value == "<<SUPPRESSED>>") {suppressed} else {}">@sourceValue.value</span>
```

Comment:
I wonder if `ConfigSourceValue` could be an ADT, or at least have a field to make whether it's been suppressed or not?